docs/plugins.rst

```diff
@@ -58,6 +59,10 @@ The temporary directory for the Girder Worker task is mapped into the running co
 under the directory ``/data``, so any files that were fetched into that temp directory
 will be available inside the running container at that path.
 
+By default, the image you specify will be pulled using the ``docker pull`` command.
+In some cases, you may not want to perform a pull, and instead want to rely on the
+image already being present on the worker system. If so, set ``pull_image`` to false.
```

Comment:
I don't know enough about what you're talking about to add that to the docs, but if you add a sentence or two about that it would be cool :)